routes/sapiV1CapiV2.js

```diff
@@ -0,0 +1,133 @@
+const express = require('express');
+const router = express.Router();
+const sapiV1CapiV2 = require('../lib/sapiV1CapiV2');
+const debug = require('debug')('views:sapiV1CapiV2');
+const bodyParser = require('body-parser');
```

Comment:
Body parser shouldn't be needed here since it's declared in the index file.

Reply:
done. left as a comment.